src/main/index.ts

```diff
@@ -68,6 +67,7 @@ import * as Extensions from "./Extensions";
     Core.DialogModule.bootstrap(dependencyRegistry);
     Core.TerminalModule.bootstrap(dependencyRegistry);
     Core.ExtensionRegistryModule.bootstrap(dependencyRegistry);
+    Core.DockModule.bootstrap(dependencyRegistry);
```

Comment:
Why did you move it so far down? As far as I can see the Dock module just depends on `App`, `SettingsManager` and `EventSubscriber`. I suggest bootstrapping it right after `SettingsManager`, which is the required dependency that is bootstrapped last.

Reply:
Right, makes sense. Haven't thought about this. Thanks for the suggestion